src/redux/reducers/experimentSettings/processingConfig/discardChangedQCFilters.js

```diff
@@ -1,9 +1,15 @@
 /* eslint-disable no-param-reassign */
-import produce from 'immer';
+import produce, { current } from 'immer';
 
 import initialState from '../initialState';
 
 const discardChangedQCFilters = produce((draft) => {
+  const originalProcessing = current(draft.originalProcessing);
```

Comment:
What is the difference between `current` and `original`?

Reply:
from what I understood `current` takes a copy of the current state of the draft, while `original` takes the original state